Source/Controller/MainViewController.swift

```diff
@@ -140,6 +153,10 @@ class MainViewController: UITabBarController {
         self.setViewControllers(controllers, animated: false)
     }
 
+    required init?(coder: NSCoder) {
```

Comment:
Can you mark this with @unavailable since it shouldn't ever be called?

Reply:
👍